### ISSUE TITLE
nothing happens when an asset is not within a group

### ISSUE THREAD

**Issue by rezaali** [jacobmoncur/SketchVectorDrawable#7]
Thank you for this! I found that it only worked for me after I "grouped" my asset and then used your plugin! 

Cheers, 
Reza

**Comment by vibin** [comment]
Yeah...worked for me for groups, but not Artboards! Thanks for the plugin!

**Comment by c5inco** [comment]
This should definitely support Artboards...
